src/interactive/userunique.py

```diff
@@ -0,0 +1,108 @@
+import logging
+
+from typing import Dict, Tuple
+
+import discord
+
+from interactive.selection import TimedView
+
+logger = logging.getLogger(__name__)
+
+
+class PromptModal(discord.ui.Modal, title="e-bot"):
+    def __init__(self, content):
+        super().__init__()
+        self.response = discord.ui.TextInput(label=content)
```

Comment:
May want to try out different text input types. The default one seems to remember your inputs (at least on discord browser) which might get a bit irritating after many games. Could be the paragraph text input is more amnesiac?